focus-areas/people/contributors.md

```diff
@@ -1,75 +1,98 @@
-  * Column headings: Never, Rarely(less than once a month), Sometimes (more than once a month), Often(once a week or more)
-  * Rows include: a) Contributing/reviewing code, b) Creating or maintaining documentation, c) Translating documentation, d) Participating in decision making about the project’s development, e) Serving as a community organizer, f) Mentoring other contributors, g) Attending events in person, h) Participating through school or university computing programs, i) Participating through a program like Outreachy, Google Summer of Code, etc., j) Helping with the ASF operations (e.g., board meetings or fundraising) 
+# **Contributors**
+
+**Question:** Who are the contributors to a project?
+
+## **Overview**
+A contributor is defined as anyone who contributes to the project in any way. This metric ensures that all types of contributions are fully recognized within the project. Identifying contributors helps the community recognize and celebrate various contributions, from code development to event planning and marketing efforts.
+
+Recognizing the contributors is vital to understanding who is driving different activities and informs project health by showing diversity and breadth in project involvement.
+
+## **Want to Know More?**
+
+<span markdown="1"><details>
+<summary>Click to read more about this metric.</summary>
+
+### **Data Collection Strategies**
+- **Aggregators:** Collect contributor names from collaboration tools, such as source code repositories, issue trackers, event registrations, mailing lists or any other tools a project uses.
+   - *Count:* Total number of contributors during a given time period.
+- **Parameters:**
+   - *Period of time:* Start and end dates for the period in which contributions are counted (default: forever).
+- **Data Sources:** Contributor data can be collected from collaboration platforms like GitHub, GitLab, IRC, blogs, and forums, or from tools like GrimoireLab and Augur.
+
+### **Filters**
+Contribution data can be filtered in several ways, such as:
+- By the location of engagement:
+   - Commit authors
+   - Issue authors
+   - Review participants (e.g., pull requests)
+   - Mailing list authors
+   - Event participants
+   - IRC or forum authors
+- By project characteristics:
+   - By release cycle
+   - Programming languages of the project
+   - Role or function in the project (e.g., coding, organizing, mentoring)
+   - Timeframe of activity (e.g., identifying new contributors)
+
+### **Visualizations**
+1. **List of Contributors:**  
+   Display contributor names with engagement details.
+<img width="490" alt="IMG_5268" src="https://github.com/user-attachments/assets/456864b5-6632-4fda-93a6-0ada9ca10b45">
+ 
+  *Figure 1: List of contributors and engagement (Source, Year)*
+
+2. **Summary of Contributors:**  
+   A summary showing the total number of contributors.
+![IMG_5269](https://github.com/user-attachments/assets/1d60591c-3024-4068-82ed-241846c03ac7)
```

Comment:
@yigakpoa can you use the link as it was in the original file?

Reply:
Done, Thank you Elizabeth